File: src/domain/IncidentManagement.Domain/TimelineEvents/TimelineEventType.cs
namespace IncidentManagement.Domain.TimelineEvents;

public enum TimelineEventType
{
    StatusChanged = 1,
    SeverityChanged = 2,
    OwnerAssigned = 3,
    Acknowledged = 4,
    CommentPosted = 5,
    SystemEvent = 6
}

File: Domain/TimelineEvent/Model/TimelineEventType.php
<?php

declare(strict_types=1);

namespace Domain\TimelineEvent\Model;

enum TimelineEventType: string
{
    case StatusChanged = 'StatusChanged';
    case SeverityChanged = 'SeverityChanged';
    case OwnerAssigned = 'OwnerAssigned';
    case Acknowledged = 'Acknowledged';
    case CommentPosted = 'CommentPosted';
    case SystemEvent = 'SystemEvent';
}